pkgs/development/python-modules/dicom2nifti/default.nix

```diff
@@ -1,26 +1,31 @@
 {
-  lib,
   buildPythonPackage,
   fetchFromGitHub,
-  pythonOlder,
-  pytestCheckHook,
+  fetchpatch,
+  lib,
+
+  # build-system
+  setuptools,
+
+  # dependencies
   gdcm,
   nibabel,
   numpy,
+  pillow,
   pydicom,
-  pylibjpeg,
-  pylibjpeg-libjpeg,
   scipy,
-  setuptools,
+
+  # tests
+  pylibjpeg-libjpeg,
+  pylibjpeg,
+  pytestCheckHook,
 }:
 
 buildPythonPackage rec {
```

Comment:
I personally don't mind, but another reviewer may ask you to remove `rec`, and to extract everything you refer to recursively into a `let / in` binding.
This seems to only affect `pname` and `version`; you would pass them to `buildPythonPackage` via `inherit`. 
The `fetchFromGitHub` call should continue to work.

Reply:
I dont understand why would we do that tho? it is more unreadable that way. also the `rec` keyword is in almost every package. And all the exceptions i see to this have relatively more complex logic in them that justifies that.

If you dont mind, i am against this change.

Reply:
That is true :) Let's leave it like this then; the output of a successful `nixpkgs-review` run would suffice for my approval.